docs/content/connector/connector_protocol.md

```diff
@@ -0,0 +1,657 @@
+When an installation is removed a DELETE request is send against *installationCallbackURL/-installationId-*.
+
+### New installation
+
+```http
+POST /-installationCallbackURL- HTTP/1.1
+Content-Type: application/json
+Signature: -base64encodedSignature-
+
+{
+   "id":"-installationID-",
+   "token":"-installationToken-",
+   "state":1,
+   "configuration":[
+      {
+         "id":"-installationParameter-",
+         "value":"-parameterValue-"
+      }
+   ]
+}
+
+Note: body was formatted for better readability
+```
+
+Note, that the request headers contain a signature.
+Every request from the connctd platform will contain such a signature.
+The signature must be verified by using the connector publication public key to verify the authenticity of the request.
+Requests without a valid signature must be ignored, since it can not be guaranteed that those requests originate from the connctd platform.
+The connector publication public key is shown once during connector publication, encoded as Base64.
+In order to use it to verify a signature it must be decoded first.
+
+Our [SDK connector-go](https://github.com/connctd/connector-go) provides a handler that verifies the signature of incoming requests.
+If you would like to implement the verification on your own, the signature is created over the following canonicalized representation of the request:
+
+`(method):-method-\r\n(url):-scheme-://-host--requestURI-\r\n(Date):-dateHeader-\r\n(body):-requestBody-`
+
+You can also consult our [reference implementation](https://github.com/connctd/api-go/blob/main/crypto/signing.go) and [SDK](https://github.com/connctd/connector-go/blob/40807b6937086d5b1755368f0141e8ce27a22754/handlers.go#L24) for more details on how to implement the signature verification.
+
+<!-- TODO: The private key (and in case of ed25519 thereby the public key) is stored in vault. We could restore the pub key, in case a connector developer looses it -->
+
+#### Payload
+
+`id` is the ID of the new installation generated by the connctd platform.
+It is used for further communication with the platform and should be stored by the connector.
+
+`token` is the ***installation token***, that is generated by the connctd platform for this specific installation that must be used for all request regarding this installation.
+The connector should store it to be able to communicate with the connctd platform about this installation.
+
+`state` is the current state of the installation.
+It is initially set to **1** (**INITIALIZED**) and can have one of the values described [below](#installation-states).
+
+`configuration` will contain all configuration parameters provided by the installation.
+It will contain at least the required parameters and all parameters are type checked and validated according to the requirements specified in the connector publication before sending the request.
```

Comment:
```suggestion
It will contain at least the required parameters. Additionally all parameters are type checked and validated beforehand  according to the requirements specified in the connector publication.
```